### PR TITLE
LPS-118120 baseline

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java [hunk 1]
```diff
@@ -178,6 +224,27 @@ protected int doCountByKeywords(
 			inlineSQLHelper);
 	}
 
+	protected int doCountByKeywords(
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java:227]: @linolaoj, this method is duplicating code when compared with the existing method `doCountByKeywords` just because of the new parameter `status`. Can you please re-use the existing logic instead and do something similar too what is being done on method `addFormInstance`? The method `addFormInstance` has a single logic shared among all methods named `addFormInstance`. Please, check if it's necessary do this same change on other parts of this pull. 😉 

Thanks.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/custom-sql/default.xml [hunk 2]
```diff
@@ -83,6 +100,34 @@
...
+						(DDMFormInstance.formInstanceId = DDMFormInstanceVersion.formInstanceId) AND
+						(DDMFormInstance.companyId = ?) AND
+						(DDMFormInstance.groupId = ?) AND
+						(DDMFormInstanceVersion.status = ?) AND
+						(
+							(LOWER(DDMFormInstance.name) LIKE ? [$AND_OR_NULL_CHECK$]) [$AND_OR_CONNECTOR$]
+							(DDMFormInstance.description LIKE ? [$AND_OR_NULL_CHECK$])
+						)
+				)
+				TEMP_TABLE
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/custom-sql/default.xml:123]: @linolaoj, can you please clarify why did you need to use a `TEMP_TABLE` here?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/configuration.jsp [hunk 3]
```diff
@@ -56,7 +56,7 @@ DDMFormInstance selFormInstance = DDMFormInstanceServiceUtil.fetchFormInstance(f
 						<liferay-ui:search-container
 							emptyResultsMessage="no-forms-were-found"
 							iteratorURL="<%= configurationRenderURL %>"
-							total="<%= DDMFormInstanceServiceUtil.searchCount(company.getCompanyId(), scopeGroupId, keywords) %>"
+							total="<%= DDMFormInstanceServiceUtil.searchCount(company.getCompanyId(), scopeGroupId, keywords, 0) %>"
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/configuration.jsp:59]: @linolaoj, please use the constant `WorkflowConstants.STATUS_APPROVED` instead.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/custom-sql/default.xml (hunk 2 above)
Comment (linolaoj) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/custom-sql/default.xml:123]: @natocesarrego , this query is supposed to be the same of com.liferay.dynamic.data.mapping.service.persistence.DDMFormInstanceFinder.findByC_G_N_D but including the DDMFormInstanceVersion table to check the status. I just didn't want to change the original query.

**Comment by linolaoj** [comment]
resent at https://github.com/natocesarrego/liferay-portal/pull/745
